roles/container-engine/containerd-common/tasks/main.yml

```diff
@@ -17,3 +17,11 @@
       skip: true
   tags:
     - facts
+
+- name: install container-selinux
+  package:
+    name: container-selinux
```

Comment:
Is it still necessary to install libseccomp/libseccomp2 for yum/apt 🤔

Reply:
`container-selinux` is not libsecomp, it is a bunch of selinux modules built for generic container hosts and is needed in case selinux is enabled.

Reply:
Actually I think you were right, about ensuring libseccomp is present, but this is separate from the need to have `container-selinux` present and is not required by `container-selinux` so I added it to `containerd-common` to ensure it gets installed properly.